docs/userstylesyml.md

```diff
@@ -0,0 +1,143 @@
+<p align="center">
+  <h2 align="center">📖 <code>userstyles.yml</code></h2>
+</p>
+
+<p align="center">
+	A guide to all things <a href="../scripts/userstyles.yml"><code>userstyles.yml</code></a>.
+</p>
+
+&nbsp;
+
+### Table of Contents
+
+<!--toc:start-->
+- [Adding a new userstyle](#adding-a-new-userstyle)
+- [Adding yourself as a maintainer](#adding-yourself-as-a-maintainer)
+- [Removing yourself as a maintainer](#removing-yourself-as-a-maintainer)
+<!--toc:end-->
+
+### Adding a new userstyle
+> [!NOTE]
+> The READMEs for each port (and many other areas of this repository) are auto-generated by scripts in [`scripts/generate/`](../scripts/generate). The [`userstyles.yml`](../scripts/userstyles.yml) file is responsible for holding the necessary data for the automated workflows.
```

Comment:
Maybe this note could be converted into an introduction at the top of this document?